changelog/pr6795.dd

```diff
@@ -0,0 +1,31 @@
+Fix Issue 10706: std.algorithm.setops functions require SortedRange arguments.
+
+The functions
+
+* largestPartialIntersection
+* largestPartialIntersectionWeighted
+* multiwayMerge
+* multiwayUnion
+* setDifference
+* setIntersection
+* setSymmetricDifference
```

Comment:
This is ddoc, not markdown. See DAutolist. Until ddoc doesn't support markdown, you'll have to use `$(UL)` and `$(LI)`

Reply:
I knew that at some point.
Should be fixed now